CuratedContent/FiveSelfishReasonsToWorkReproducibly.md

```diff
@@ -0,0 +1,33 @@
+# Five Selfish Reasons to Work Reproducibly
+<!--deck text start-->
+How can working reproducibly help you? This article is a quick read, but you may even want to revisit this article occasionally to remind yourself that ensuring your work is reproducible pays off in the long run for you.
+
+<!--deck text end-->
+
+#### Contributed by [Patricia Grubel](https://github.com/pagrubel "Patricia Grubel")
+#### Publication date: January 24, 2025
+
+Resource information | Details
+:--- | :---
+Article Title | [Five Selfish Reasons to Work Reproducibly](https://genomebiology.biomedcentral.com/articles/10.1186/s13059-015-0850-7)
+Authors | Florian Markowetz
+Focus | Reproducibility
+
+Are you selfish enough? Many conferences and sponsors are requiring proof of reproducibility, driving scientific software developers to work reproducibly, but how does the same idea benefit you as a scientist or developer?
+
+The author presents five reasons to work reproducibly and discusses the implications of reproducible work as a scientific researcher.  There are explanations of why working reproducibly is beneficial to you and your career.
+Although the author presents the ideas in the context of scientific research, the ideas are applicable to scientific software development as well.
+The five reasons to work reproducibly are to avoid disaster, make it easier to write papers, help reviewers see it your way, enable continuity of your work and to help build your reputation.
```

Comment:
I would suggest moving up this list just after the first sentence and use bullets, like:

----

The author presents **five reasons to work reproducibly:**

* **avoid disaster**,
* **make it easier to write papers**,
* **help reviewers see it your way**,
* **enable continuity of your work**, and
* **help to build your reputation**

and discusses the implications of reproducible work as a scientific researcher. There are explanations of why working reproducibly is beneficial to you and your career. ...

Reply:
I actually started with a bulleted list and will go back to that.